Core/Core/Transports/TransportExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Speckle.Core.Transports;
+
+public static class TransportExtensions
+{
+  /// <summary>
+  /// <inheritdoc cref="ITransport.SaveObject(string, string)"/>
+  /// Retrieves the object from the provided transport
+  /// </summary>
+  /// <param name="destination">The destination</param>
+  /// <param name="id"><inheritdoc cref="ITransport.SaveObject(string, string)"/></param>
+  /// <param name="source">The transport from where to retrieve it.</param>
+  /// <exception cref="TransportException">Failed to save object</exception>
+  /// <exception cref="System.OperationCanceledException"><see cref="System.Threading.CancellationToken"/> requested cancel</exception>
+  public static void SaveObject(this ITransport destination, string id, ITransport source)
```

Comment:
All our transports implemented this function the same. So I've made it a helper function, rather than included in the `ITransport`. This should be a non-source breaking change (since its in the same namespace)

Reply:
Oh, you learn something new every day, i never tried to extend an interface 😅

Reply:
Forgot to mention here, i added a comment about this in the deleted interface method before realising this was here!